### ISSUE TITLE
No call events for newly created lines

### ISSUE THREAD

**Issue by portison** [markjulmar/atapi.net#11]
Hi, This issue was originally submitted on Codeplex, but I see it is still an issue in the latest version on GitHub. This issue had me scratching my head for a while back in 2016. The suggested change below is now part of my standard ATAPI build and running live with a number of different phone systems - although we have only found it to be a major issue on Cisco CUCM (tested with CUCM 9.x, 10.x and 11.x) with extension mobility.

Original Codeplex Issue Text below:-

Lines added after the initial ATAPI initialisation do not fire any call events, although the lines are controllable.

The fix for this issue is a code change to register the TapiManager event handlers immediately following creation of the new TAPILine in TapiManager.ProcessTapiMessage. 

This is tested against CUCM 9.1 where lines are removed and added when mobility users log-out and log-in to handsets.

TapiManager.cs:

```` C#
case TapiEvent.LINE_CREATE:
{
	var newLine = new TapiLine(this, msg.dwParam1.ToInt32());

	// Change to add event handler registration
	newLine.NewCall += HandleNewCall;
	newLine.CallStateChanged += HandleCallStateChanged;
	newLine.CallInfoChanged += HandleCallInfoChanged;
	newLine.AddressChanged += HandleAddressChanged;
	newLine.Changed += HandleLineChanged;
	newLine.Ringing += HandleLineRinging;
	// End of change

	lock (_lineArray)
	{
	_lineArray.Add(newLine);
	}
	if (LineAdded != null)
	LineAdded(this, new LineAddedEventArgs(newLine));
}
break;

````

**Comment by markjulmar** [comment]
Great! Can you submit a pull request? We can integrate it pretty easily that way

**Comment by portison** [comment]
Not being a previous git user, I'm trying to do this all on the Github website. Forking was ok, but the minor edit to the cs file looks to have replaced all the line breaks in the file.  I've got as far as pressing the create pull request button for the fork - but that just gives me a page with an enormous textbox that I cannot type into. 

Creating a push request on GitHub is becoming more of a challenge than identifying the atapi event handler issue. I'll have another go in the morning.  :-)

**Comment by markjulmar** [comment]
Fixed with PR #12.
